src/bridge/bridge.c

```diff
@@ -164,6 +165,10 @@ send_init_command (CockpitTransport *transport,
             json_object_set_object_member (object, "os-release", block);
           g_hash_table_unref (os_release);
         }
+
+      session_id = getenv ("XDG_SESSION_ID");
```

Comment:
Let's please use  `secure_getenv()`. It Should Not make a Difference™ for most scenarios, but I'm not sure I have every corner case in my head about this.

Reply:
Done.